src/test/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/integration/IntegrationTestBase.kt

```diff
@@ -24,12 +24,12 @@ abstract class IntegrationTestBase {
 
   internal fun setAuthorisation(
     username: String? = "AUTH_ADM",
-    roles: List<String> = listOf("ROLE_EM_CEMO__CREATE_ORDER"),
+    roles: List<String> = listOf("ROLE_EM_CEMO__CREATE_ORDER", "ROLE_EM_CEMO__CREATE_DEVICE_WEARER"),
```

Comment:
I don't think we need a role for each entity. I think the role we have is the same as saying "create the whole order" rather than "create the order entity".